openedx/features/badging/handlers.py

```diff
@@ -26,3 +31,18 @@ def delete_badge_info_from_nodebb(sender, instance, **kwargs):
         'id': instance.id
     }
     task_delete_badge_info_from_nodebb.delay(badge_data)
+
+
+@receiver(startup.perform_type_registrations)
+def register_notification_types(sender, **kwargs):  # pylint: disable=unused-argument
+    """
+    Register philu NotificationTypes.
+    This will be called automatically on the Notification subsystem startup (because we are
+    receiving the 'perform_type_registrations' signal)
```

Comment:
Lets remove `(because we are receiving the 'perform_type_registrations' signal)`, it is redundant because decorate is self-explanatory

Reply:
done